packages/builder/src/definition.ts

```diff
@@ -292,11 +293,26 @@ export class ChainDefinition {
     // it would be best if the dep was downloaded when it was discovered to be needed, but there is not a lot we
     // can do about this right now
     return _.uniq(
-      Object.values(this.raw.import).map((d) => ({
-        source: _.template(d.source)(ctx),
-        chainId: d.chainId || ctx.chainId,
-        preset: _.template(d.preset || 'main')(ctx),
-      }))
+      Object.values(this.raw.import).map((d) => {
```

Comment:
Not sure these changes are even relevant or correct, the getRequiredImports function doesnt even get called anywhere else.